diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/CommentsFormatting.kt

```diff
@@ -106,16 +109,16 @@ class CommentsFormatting(private val configRules: List<RulesConfig>) : Rule("kdo
         } else if (node.treeParent.elementType != IF) {
             checkClassComment(node)
         }
-        checkWhiteSpaceBeforeComment(node, configuration)
+        checkWhiteSpaceBeginInComment(node, configuration)
     }
 
     private fun handleEolAndBlockComments(node: ASTNode, configuration: CommentsFormattingConfiguration) {
-        basicCommentsChecks(node, configuration)
-        checkWhiteSpaceBeforeComment(node, configuration)
+        if (node.treeParent.elementType != FILE)  basicCommentsChecks(node, configuration)
```

Comment:
What will happen if there is a top-level property with comment after it? It won't be checked? Like
```kotlin
package org.cqfn.diktat

val foo = 42// comment that has invalid spacing
```

Reply:
No, in this case comment belongs to property. But you are right about top-level, I fixed it.